### ISSUE TITLE
Inference result on lingbot-va-base

### ISSUE THREAD

**Issue by liu-ht23** [Robbyant/lingbot-va#23]
Thanks for the great work!

I used the lingbot-va-base checkpoint and ran the inference using the va_franka_i2va config, with the prompt "pick up the yellow bread", but got the result below. 

I don't know if I got anything wrong, I checked my transformers version is 4.55.2. I also noticed some weight is not loaded from the checkpoint and I don't know if it is normal.

<img width="1253" height="77" alt="Image" src="https://github.com/user-attachments/assets/c3f5215a-7444-42a8-8ef4-676afd81ea10" />

https://github.com/user-attachments/assets/9ef767e7-ce8d-4d39-8f0d-b4b1d8e2d569

**Comment by lilin-hitcrt** [comment]
This is actually expected behavior. The data in this example was collected from our own physical robot setup, which is completely unseen in the pretraining dataset. That's why the base model doesn't generate good visualizations on it. This was a test case we forgot to remove when organizing the open-source code. We'll replace this example with data from the pretraining validation set in the future.

If you want to see the video generation quality, I'd suggest running the RobotWin experiment instead.
